client/my-sites/checkout/composite-checkout/components/checkout-help-link.tsx

```diff
@@ -193,36 +198,50 @@ export default function CheckoutHelpLink() {
 
 	const hasDirectSupport = supportVariation !== SUPPORT_FORUM;
 
+	const getZendeskChatWidget = () => {
+		if ( isAkismetCheckout() ) {
+			return <ZendeskJetpackChat keyType="akismet" />;
+		}
+
+		if ( isJetpackCheckout() ) {
+			return <ZendeskJetpackChat keyType="jpCheckout" />;
+		}
+
+		if ( isPresalesZendeskChatEligible ) {
```

Comment:
I believe this check is what's stopping presales chats from loading for logged-out users.

When testing, the Akismet and Jetpack chats are loading while logged-out - is that what we want?

Reply:
Oh good catch, I don't believe we want to show this for logged out users, but I will ask just to be safe

Reply:
Looks like we do want to show it to logged out users: p1683154661719209/1683123825.685809-slack-C02LK1W8T4Z